### PR TITLE
Mods "[Aftershock]Adjustment of overgenerated facilities"

### PR DESCRIPTION
**Pull request by hagetawashi**
#### Summary
Mods "[Aftershock]Adjustment of overgenerated facilities"

#### Purpose of change
If aftershock is implemented and aftershock_exoplanet is not implemented, we believe that the creation of some facilities is inhibited by overgenerated malls or large abandoned ruins.
I also found a very large number of office tower laboratories, which I saw as a problem as well.

#### Describe the solution
I have adjusted the values of "region_settings" and "mutable_ruinfield" enclosed in the aftershock and added "UNIQUE" flag to "mutable_ruinfield".
Also, adjusted the value of "region_settings" enclosed in the aftershock_exoplanet and removed the "UNIQUE" flag from "mutable_ruinfield".

#### Describe alternatives you've considered
This modification significantly reduces the incidence of malls when played with aftershock_exoplanet installed.
To solve this problem, it is necessary to add a description about malls in region_settings of aftershock_exoplanet.
Also, the values set in this fix may not be appropriate.
In line with this modification, changes were made to ensure that the generation rate in aftershock_exoplanet is the same as before.

#### Testing
Worlds were generated for each of the 100 floors before and after the modification, and the number of times one or more was generated was determined. The average number generated is shown in parentheses.

Before modification
mall: 88% (2.14)
office tower laboratories: 86% (1.93)
ruins: 100%(2)

After modification
mall: 30%(0.31)
office tower laboratories: 040%(0.51)
ruins: 14%(0.14)

#### Additional context
This is my first time sending a pull request with github.
I have tried to make it as official as possible, but I may have made some mistakes. If so, I would appreciate it if you could let me know.

### CODE REVIEW

**Comment by John-Candlebury** [comment]
The mall change is alright. 

The ruins one isn't.  Its the only reliable place to get aftershock content that isn't crafting scrap or CBMs. A pair of ruined structures every overmap out in the wilderness that arent super hard to find is necessary for people to experience the stuff the mod actually adds.

**Comment by hagetawashi** [comment]
> The mall change and city generation changes are alright.
> 
> The ruins one isn't. Its the only reliable place to get aftershock content that isn't crafting scrap or CBMs. A pair of ruined structures every overmap out in the wilderness that arent super hard to find is necessary for people to experience the stuff the mod actually adds.

I understand regarding the handling of the ruins.
I have just noticed a flaw in my pull request. Before I make any corrections, let me discuss this with you to resolve them.
In my pull request, I mentioned malls, labs, and ruins, but did not mention the other city adjustments. I think this was not sincere and I regret it.
Should i undo the value adjustments made to the other cities, such as hospitals, and make a new pull request for this?
It would be very helpful if you could let me know.

**Comment by John-Candlebury** [comment]
Np you can keep the entirety of the city changes here.

**Comment by hagetawashi** [comment]
Okay, thank you very much.
I will start working on it now.

**Comment by hagetawashi** [comment]
I have made the corrections to the areas you mentioned and removed the additional files that are no longer needed accordingly.
I think this will work fine.

File: data/mods/aftershock_exoplanet/region_settings.json [hunk 1]
```diff
@@ -35,7 +35,16 @@
...
+      "shops": {
+        "afs_augmentation_clinic_1": 400,
+        "city_block_2": 700,
+        "urban_35_hospital": 875,
+        "office_skyscraper": 400,
+        "office_tower_large": 400,
+        "office_tower_hiddenlab": 800,
+        "afs_astrobiology_lab": 300,
+        "mall": 700
+      }
```
Comment (John-Candlebury) [inline data/mods/aftershock_exoplanet/region_settings.json:47]: I think these changes got added here by mistake, you should keep the orignial list.

Should be ready for merging after that.

Comment (hagetawashi) [reply data/mods/aftershock_exoplanet/region_settings.json]: I have restored the points you raised and would appreciate your confirmation.

**Comment by John-Candlebury** [comment]
Thanks for the help!

**Pull request merged**